build/charts/theia/templates/grafana/deployment.yaml

```diff
@@ -103,8 +21,8 @@ spec:
           - 0
       containers:
         - name: grafana
-          image: projects.registry.vmware.com/antrea/theia-grafana:8.3.3
-          imagePullPolicy: IfNotPresent
+          image: {{ .Values.grafana.image.repository }}:{{ .Values.grafana.image.tag }}
+          imagePullPolicy: {{ .Values.grafana.image.pullPolicy }}
           env:
             - name: GF_INSTALL_PLUGINS
               value: "https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.0.zip;theia-grafana-sankey-plugin,grafana-clickhouse-datasource 1.0.1"
```

Comment:
shall we add the value of `GF_INSTALL_PLUGINS` to values.yaml?

Reply:
Added.